outputs/discord.go

```diff
@@ -0,0 +1,30 @@
+package outputs
+
+import (
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+type discordPayload struct {
+	Content    string `json:"content"`
+	Avatar_url string `json:"avatar_url,omitempty"`
+}
+
+func newDiscordPayload(falcopayload types.FalcoPayload, config *types.Configuration) discordPayload {
+	iconURL := "https://raw.githubusercontent.com/falcosecurity/falcosidekick/master/imgs/falcosidekick.png"
```

Comment:
It should be a configurable option like for `Slack`.

Reply:
ARe you looking for a binary "custom avatar" = yes/no or a configurable avatar?

Reply:
A configurable avatar, and this as default.